src/hosting/api.ts

```diff
@@ -432,9 +434,17 @@ export async function getCleanDomains(project: string, site: string): Promise<st
  * updates Firebase Auth Api with aforementioned
  * list.
  * @param project the project ID.
- * @param site the site for the channel.
+ * @param sites the list of sites for the channel.
  */
-export async function cleanAuthState(project: string, site: string): Promise<string[]> {
-  const authDomains = await getCleanDomains(project, site);
-  return await updateAuthDomains(project, authDomains);
+export async function cleanAuthState(
+  project: string,
+  sites: string[]
+): Promise<Map<string, Array<string>>> {
+  let siteDomainMap = new Map();
```

Comment:
```suggestion
  const siteDomainMap = new Map();
```

Good use of Map too :)